crypto/ml_kem/ml_kem_ipd_ref_common/indcpa.h

```diff
@@ -6,22 +6,25 @@
 #include "polyvec.h"
 
 #define gen_matrix KYBER_NAMESPACE(gen_matrix)
-void gen_matrix(polyvec *a, const uint8_t seed[KYBER_SYMBYTES], int transposed);
```

Comment:
+1 to sanketh's earlier comment -- seed size is invariant over parameters, right? why can't we define this as an array rather than a pointer?

Reply:
yes, will fix in https://github.com/aws/aws-lc/pull/1775